3rdparty/CMakeLists.txt

```diff
@@ -1,7 +1,25 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-# Note: order does matter
+if (WIN32)
+  # NOTE: On Windows we have found that we must use Git Bash, not the
+  # Bash from the Windows Subsystem for Linux. Hence this is
+  # explicitly searching only for Git Bash. See #1302 for more.
+  find_program(GIT git)
+  get_filename_component(GIT_DIR ${GIT} DIRECTORY)
+  find_program(BASH bash
+    PATHS "C:/Program Files/Git/bin" "${GIT_DIR}/../bin"
+    NO_DEFAULT_PATH) # Do not find WSL bash.
+
+  if (NOT BASH)
+    message(FATAL_ERROR "Git Bash not found!")
```

Comment:
Maybe change the error message to include some rectification action the user can take.